application/render/PageBuilder.php

```diff
@@ -149,6 +149,8 @@ private function initialize()
 
         $this->tpl->assign('formatter', $this->conf->get('formatter', 'default'));
 
+        $this->tpl->assign('links_per_page', $_SESSION['LINKS_PER_PAGE']);
```

Comment:
This class hasn't been entirely refactored yet, but for new code we try to avoid accessing global variables.

```suggestion
        $this->tpl->assign('links_per_page', $this->session['LINKS_PER_PAGE']);
```